ui/e2e/instances/details/index.spec.ts

```diff
@@ -87,3 +95,156 @@ test("the header of the instance page shows the relevant data for the workflow",
     "the button 'Open Workflow' is clickable and links to the correct URL"
   ).toHaveURL(editURL);
 });
+
+test("the diagram panel on the instance page responds to user interaction", async ({
+  page,
+}) => {
+  const newInstance = createInstance({ namespace, path: simpleWorkflowName });
+  await expect(newInstance, "wait until process was completed").toBeDefined();
+  const instanceId = (await newInstance).instance;
+  await page.goto(`/${namespace}/instances/${instanceId}`);
+
+  const diagramPanel = page.getByTestId("rf__wrapper");
```

Comment:
I don't understand this one. Can you clarify?
I have declared the same variable again in the second test, because I can't access the variable from the first test.

Reply:
Sorry, I missed that. My suggestion was the following (but the code that this was about was deleted anyways)

to change this
```typescript
const diagramPanel = page.getByTestId("rf__wrapper");
(...)
// hover over something else to make the overlay disappear
page.getByTestId("rf__wrapper").hover();
(...)
// hover over something else to make the overlay disappear
page.getByTestId("rf__wrapper").hover();
```

to 

```typescript
const diagramPanel = page.getByTestId("rf__wrapper");
(...)
// hover over something else to make the overlay disappear
diagramPanel.hover();
(...)
// hover over something else to make the overlay disappear
diagramPanel.hover();
```

Reply:
Aha! 💡 
Sorry, I probably didn't start from the top with my changes. Yes, that would have made sense, but as you pointed out, we don't need that extra hover.